airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -984,10 +982,12 @@ class CRMSearchStream(IncrementalStream, ABC):
     updated_at_field = "updatedAt"
     last_modified_field: str = None
     associations: List[str] = None
+    fully_qualified_name: str = None
 
     @property
     def url(self):
-        return f"/crm/v3/objects/{self.entity}/search" if self.state else f"/crm/v3/objects/{self.entity}"
+        object_type_id = self.fully_qualified_name or self.entity
```

Comment:
I'm curious, is the `fully_qualified_name` supposed to be a replacement of the `entity/name`? What I'm wondering is if we even need to pass the non-qualified name around any more? It would simplify this logic and we could actually continue using the name -> schema map we originally had too.

I guess it comes down to whether we can rely on the `fully_qualified_name` always being present in the response. Do you know if that is the case?

Reply:
I wouldn't rely on the fullyQualifiedName in all cases. As for now, it is only used in the CustomObject class, and I'd like to not make its usage more widespread unless there's a reasonable need. Hubspot's documentation is very tricky, so I prefer to make less changes if possible

Reply:
fair enough, thanks for clarifying!